mods/seasonal/random-teams.js

```diff
@@ -19,6 +19,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: "Pilfer",
 				evs: {hp:252, atk: 252, spe:4}, nature: 'Adamant',
 			},
+			'antemortem': {
+				species: 'Clefable', ability:'Multiscale', item: 'Leftovers', gender: 'M', //ask gender
```

Comment:
looks like ante's ability should be sheer force or multiscale, also missing a space after the `:`